src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Test\Services\SharedStorageInterface;
+use Sylius\Component\Order\Model\OrderInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\Component\User\Model\UserInterface;
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+final class OrderContext implements Context
```

Comment:
After https://github.com/Sylius/Sylius/pull/4175/files#r53758240 this context will be redundant and should be removed.